css-text-4/Overview.bs

```diff
@@ -4603,8 +4456,35 @@ Breaking Rules for Letters: the 'word-break' property</h3>
 			(which uses [=spaces=] between words),
 			and is also useful for mixed-script text where CJK snippets are mixed
 			into another language that uses [=spaces=] for separation.
+
+		<dt><dfn>auto</dfn>
+		<dd>
+			This value directs the user agent to perform language-specific content analysis
+			to determine where to insert [=virtual word boundaries=].
```

Comment:
Minor, but how about saying "to determine soft wrap opportunities and virtual word boundaries"?

I suppose you mean this value has full control to determine where soft wrap opportunities are, but "determine where to insert" may be a bit ambiguous about whether it can remove exisitng soft wrap opportunities or not.